CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ https://github.com/bitcrowd/sshkit.ex/compare/v0.0.3...HEAD
 
 * Put fixes here
 
+## `0.1.0` (work in progress)
+
+https://github.com/bitcrowd/sshkit.ex/compare/v0.0.3...v0.1.0
+
+* Require Elixir 1.5+ and drop support for lower versions.
+* Fix Elixir 1.5 String deprecations.
+* Update installation instructions.
+* Introduce Mox for mocking in unit tests.
+* Improve and add unit tests.
+
+### Potentially breaking changes:
+
+* Remove `:dry_run` option for now. Planning to reintroduce at a higher level.
+
+### New features:
+
+* Put new features here
```

Comment:
Yup, but when do we do this? I thought it makes sense to do it right before we publish the version… I'd probably forget to take another look at the empty template above otherwise 🤔